tests/run-pass/intptrcast.rs

```diff
@@ -1,4 +1,7 @@
-// compile-flags: -Zmiri-seed=0000000000000000
+// compile-flags: -Zmiri-seed=0000000000000000 
```

Comment:
Same here, new spurious trailing space.

Many editors have a setting to delete trailing space, try finding that for yours. :)